python/tvm/meta_schedule/search_strategy/evolutionary_search.py

```diff
@@ -64,13 +64,13 @@ def __init__(
         *,
         num_trials_per_iter: int,
         max_trials_per_task: int,
-        population_size: int,
-        init_measured_ratio: float,
-        init_min_unmeasured: int,
-        genetic_num_iters: int,
-        genetic_mutate_prob: float,
-        genetic_max_fail_count: int,
-        eps_greedy: float,
+        population_size: int = 2048,
+        init_measured_ratio: float = 0.2,
+        init_min_unmeasured: int = 50,
+        genetic_num_iters: int = 4,
+        genetic_mutate_prob: float = 0.85,
+        genetic_max_fail_count: int = 10,
+        eps_greedy: float = 0.05,
```

Comment:
If I want to tweak these params, I need to manually call `config.create_strategy(...)`, and I don't see an easy tuning API for such case.

Reply:
Hey Masa, thanks for your feedback! I'm trying to understand this usecase you suggested here. Would you mind sharing an example for me to act on? Thanks a lot!

Reply:
I don't know how often (if ever) we want to do this, but these default parameters seem hand-picked, so naturally I wonder if tweaking these params can lead to improvement. And if I do want to do that, the current API doesn't have an easy API like `tune_relay(...)` with manually constructed `EvolutionarySearch`.

Reply:
If you never intend users to mess with these params, the new API looks fine.

Reply:
Thanks for sharing! If I read it correctly, while indeed it's less frequent, there is definitely usecase to tweak parameters. To allow this to happen, we have an extra parameter called `TuneConfig.search_strategy_config`. For example,

```python
tune_relay(
  config=TuneConfig(
    strategy="evolutionary",
    num_trials_per_iter=64,
    max_trials_per_task=20000,
    max_trials_global=20000,
    search_strategy_config = {
      "genetic_num_iters": 10,
    },
  ),
)
```

Reply:
Oh nice! I didn't notice that. `search_strategy_config` thing doesn't seem to be used in tests, can you add a trivial one (like your example above)

Reply:
Yes of course! Yeah as the principle that less frequent features should be put somewhere less significant, I simply hide it in a small config dict :-)